src/storm/templates/configmap.yaml

```diff
@@ -21,6 +21,10 @@ data:
     storm.log4j2.conf.dir: /log4j2
     storm.local.dir: /data
     storm.log.dir: /logs
+    nimbus.childopts: {{ $.Values.nimbus.childopts }}
+    supervisor.childopts: {{ $.Values.supervisor.childopts }}
+    worker.childopts: {{ $.Values.worker.childopts }}
```

Comment:
I think we should also expose `worker.heap.memory.mb`, as it's a simpler one for defining the worker heap size

Reply:
added 💪